src/scheduled-query/azext_scheduled_query/custom.py

```diff
@@ -25,27 +25,52 @@ def _build_criteria(condition, condition_query):
     return ScheduledQueryRuleCriteria(all_of=condition)
 
 
-def create_scheduled_query(client, resource_group_name, rule_name, scopes, condition, condition_query=None,
-                           disabled=False, description=None, tags=None, location=None,
-                           actions=None, severity=2, window_size='5m', evaluation_frequency='5m',
-                           target_resource_type=None, mute_actions_duration='PT30M'):
-    from .vendored_sdks.azure_mgmt_scheduled_query.models import ScheduledQueryRuleResource
+def create_scheduled_query(client,
+                           resource_group_name,
+                           rule_name,
+                           scopes,
+                           condition,
+                           action_groups=None,
+                           custom_properties=None,
+                           condition_query=None,
+                           disabled=False,
+                           description=None,
+                           tags=None,
+                           location=None,
+                           severity=2,
+                           window_size='5m',
+                           evaluation_frequency='5m',
+                           target_resource_type=None,
+                           mute_actions_duration=None,
```

Comment:
The default value of `mute_actions_duration` is changed. Please add breaking change logs in history.